### DISCUSSION TITLE
How to setup gDriveSync plugin?

### DISCUSSION THREAD

**Discussion by hugalafutro** [jayofelony/pwnagotchi#222] [Q&A] [answered]
Hi, in the code of the plugin it says [gDriveSync] /root/client_secrets.json is empty. Please RTFM!
Where is the aforementioned F Manual? I can't find any instructions how to set it up anywhere, the only thing google finds is the code itself

Thanks for any pointers, would love to have online backup, ideally nextcloud but I'll take gdrive too.

**Comment by jayofelony** [comment, marked as answer]
On my github there is a gdrivesync readme.

**Reply by hugalafutro** [reply]
thanks I found it
